tests/core/actions/test_forms.py

```diff
@@ -1298,6 +1359,64 @@ def test_extract_other_slots_with_entity(
     assert slot_values == expected_slot_values
 
 
+@pytest.mark.parametrize(
+    "entities, expected_slot_values",
+    [
+        # Two entities were extracted for `ListSlot`
+        (
+            [
+                {"entity": "topping", "value": "mushrooms"},
+                {"entity": "topping", "value": "kebab"},
+            ],
+            ["mushrooms", "kebab"],
+        ),
+        # Only one entity was extracted for `ListSlot`
+        ([{"entity": "topping", "value": "kebab"},], ["kebab"],),
+    ],
+)
+def test_extract_other_list_slot_from_entity(
+    entities: List[Dict[Text, Any]], expected_slot_values: List[Text]
+):
+    form_name = "some_form"
+    slot_name = "toppings"
+    domain = Domain.from_yaml(
+        textwrap.dedent(
+            f"""
+    version: "2.0"
+
+    slots:
+      {slot_name}:
+        type: list
+        influence_conversation: false
+
+    forms:
+      {form_name}:
+        {REQUIRED_SLOTS_KEY}:
+          {slot_name}:
+          - type: from_entity
+            entity: topping
+    """
+        )
+    )
+
+    form = FormAction(form_name, None)
+
+    tracker = DialogueStateTracker.from_events(
+        "default",
+        [
+            SlotSet(REQUESTED_SLOT, "some slot"),
+            UserUttered(
+                "bla", intent={"name": "greet", "confidence": 1.0}, entities=entities
+            ),
+            ActionExecuted(ACTION_LISTEN_NAME),
+        ],
+        slots=domain.slots,
+    )
+
+    slots = form.extract_other_slots(tracker, domain)
```

Comment:
This test appears almost identical to the one above `test_extract_requested_slot_with_list_slot` except for the different form extract methods - why is this 2nd one required?

Reply:
The first one tests the `ListSlot` extraction for the slot which was request the form - this test tests the extraction of slots which weren't requested by the form but accidentally filled by the user